src/AccessibilityInsights.Automation/LocationHelper.cs

```diff
@@ -12,6 +12,8 @@ internal class LocationHelper
     {
         private const string OutputPathEnvironmentVariableName = "A11YSCANRESULTSFOLDER";
         private const string FallbackResultsFolder = "ScanResults";
+        public const string SarifExtension = ".sarif";
```

Comment:
>. [](start = 46, length = 1)

If you remove the leading '.' in lines 15 & 16, you won't need the substring in lines 148, 164, 218, 253, and 273 of the unit tests